lib/src/services/gamepad.dart

```diff
@@ -0,0 +1,42 @@
+import "package:win32_gamepad/win32_gamepad.dart";
+
+import "service.dart";
+
+///A service to send the comtroller's activity
+///using win32_gamepad
+class GamepadService extends Service {
+  ///The connection to the controller
+  static final gamepad = Gamepad(0);
+  
+  @override
+  Future<void> init() async{ 
+    connect();
+  }
+
+  @override
+  Future<void> dispose() async{  
+    gamepad.state = GamepadState.disconnected();
+  } 
```

Comment:
```suggestion
  @override
  Future<void> dispose() async { }
```
Good intuition, but this doesn't actually do anything useful. It sets the gamepad to disconnected, sure, but the next time someone calls `update` it'll just go back to whatever it really is. `dispose` is meant for cleanup tasks, like closing files and network connections. I don't think we need anything here at all then.